src/miniconda/.devcontainer/Dockerfile

```
@@ -1,10 +1,30 @@
 FROM continuumio/miniconda3 as upstream
 
+# Temporary: Upgrade python packages due to mentioned CVEs
+# They are installed by the base image (continuumio/miniconda3) which does not have the patch.
+RUN conda install \ 
+    # https://github.com/pyca/cryptography/security/advisories/GHSA-5cpq-8wj7-hf2v
+    pyopenssl=23.2.0 \ 
+    cryptography=41.0.2 \
+    # https://github.com/advisories/GHSA-j8r2-6x86-q33q
+    requests=2.31.0
+
 # Reset and copy updated files with updated privs to keep image size down
 FROM mcr.microsoft.com/devcontainers/base:0-bullseye
```

```suggestion
FROM mcr.microsoft.com/devcontainers/base:1-bullseye
```

Looks like we missed updating this image!